snapshots/CaliburIsValidSignatureTest.json

```diff
@@ -1,8 +1,11 @@
 {
-  "isValidSignature_P256": "45287",
-  "isValidSignature_P256_typedData_notNested_safeERC1271Caller": "32773",
-  "isValidSignature_P256_withHook": "51296",
-  "isValidSignature_WebAuthnP256": "51973",
-  "isValidSignature_rootKey": "20822",
-  "isValidSignature_rootKey_typedData_notNested_safeERC1271Caller": "8304"
+  "isValidSignature_P256_personalSign": "35124",
+  "isValidSignature_P256_typedDataSign": "42515",
+  "isValidSignature_P256_typedData_notTypedDataSign_safeERC1271Caller": "295984",
```

Comment:
why is this gas so high 💀

Reply:
that seems odd..

Reply:
I wonder if its not short circuiting?

Reply:
Oh wait sorry in this case it should be going through each of the different signature verification paths, before it hits isSafeCaller, so it does make sense that its more expensive... its just quite a lot

Reply:
Can we get a gas snap for a p256 personal sign signature? it seems like in general p256 verifications are more expensive and the only other benchmark we have here with a p256 key is when it short circuits in `isValidSignature_P256_withHook` so maybe we can see what the cost is for isValidSignature_P256_personalSign.. and roughly it should be inbetween these two gas snaps

Reply:
There is a test for that in #193 , but the huge gas cost here is definitely from the P256 library. It first tries to verify using the precompile, then if that fails (it does here since it is not a valid nestedPersonalSign), it tries to verify entirely in solidity....